eth/backend.go

```diff
@@ -480,6 +480,7 @@ func (s *Ethereum) IsListening() bool                  { return true } // Always
 func (s *Ethereum) EthVersion() int                    { return int(s.protocolManager.SubProtocols[0].Version) }
 func (s *Ethereum) NetVersion() uint64                 { return s.networkID }
 func (s *Ethereum) Downloader() *downloader.Downloader { return s.protocolManager.downloader }
+func (s *Ethereum) IsSynced() bool                     { return atomic.LoadUint32(&s.protocolManager.acceptTxs) == 1 }
```

Comment:
Please rename it to `Synced()`. Go doesn't generally have the `Is` `Get` prefixes.